baseapp/baseapp.go

```diff
@@ -89,6 +89,7 @@ type BaseApp struct {
 	addrPeerFilter sdk.PeerFilter // filter peers by address and port
 	idPeerFilter   sdk.PeerFilter // filter peers by node ID
 	fauxMerkleMode bool           // if true, IAVL MountStores uses MountStoresDB for simulation speed.
+	sigverifyTx    bool           // whether to sigverify check for transaction
```

Comment:
can we precise here, as in context what it is used for.

Reply:
done